.github/workflows/node.js.yml

```diff
@@ -0,0 +1,28 @@
+# https://docs.github.com/en/actions/guides/building-and-testing-nodejs
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        # Each version of Node.js specified in the node-version array creates a job that runs the same steps.
+        # Skipped using 14.x to avoid https://github.com/npm/cli/issues/558#issuecomment-580018468
+        node-version: [16.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: node --version && npm --version
+      - run: npm ci
```

Comment:
I think at least also `prettier-check` should be executed

Reply:
Done @matedo1